subsys/ipc/ipc_service/backends/ipc_rpmsg_multi_core.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2021, Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ipc/ipc_service_backend.h>
+#include <ipc/rpmsg_multi_instance.h>
+
+#include <logging/log.h>
+#include <sys/util_macro.h>
+#include <zephyr.h>
+#include <device.h>
+
+#define DT_DRV_COMPAT ipc_rpmsg_mc
+
+LOG_MODULE_REGISTER(ipc_rpmsg_multi_instance, CONFIG_IPC_SERVICE_LOG_LEVEL);
```

Comment:
Probably should be `ipc_rpmsg_multi_core`, or is it intentionally reusing `multi_instance` logger?